pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/SegmentDeletionManager.java

```diff
@@ -99,6 +108,10 @@ public Thread newThread(Runnable runnable) {
     });
   }
 
+  public void registerSegmentDeletionListener(PinotSegmentDeletionListener pinotSegmentDeletionListener) {
```

Comment:
This is no longer needed? It will cause exception if the list is `Collections.emptyList()`

Reply:
Not needed. Removed